index.d.ts

```diff
@@ -47,9 +57,13 @@ declare class QuickLRU<KeyType, ValueType> implements Iterable<[KeyType, ValueTy
 	/**
 	Set an item.
 
+	Individual expiration of an item can be specified with the `maxAge` parameter.
+	If not specified, the global `maxAge` value will be used in case that is specified on the constructor,
+	else the item will not have expiration.
+
 	@returns The list instance.
 	*/
-	set(key: KeyType, value: ValueType): this;
+	set(key: KeyType, value: ValueType, maxAge?: number): this;
```

Comment:
Looking at the callsite of this, I don't think `set('foo', 1, 100)` is very readable. It's not clear to the person reading it what the last number is. I think it should be an options-object instead. Sorry for not catching this earlier.